persistence-cassandra/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/cassandra/CassandraReadSideSpec.scala

```diff
@@ -44,3 +44,17 @@ class CassandraReadSideSpec extends CassandraPersistenceSpec(CassandraReadSideSp
   }
 
 }
+
+class CassandraReadSideAutoCreateSpec extends CassandraPersistenceSpec(CassandraReadSideSpec.noAutoCreateConfig) {
+  import system.dispatcher
+
+  private lazy val testSession: CassandraSession = new CassandraSession(system)
+  private lazy val testCasConfigProvider: CassandraProvider = new CassandraProvider(system)
+  private lazy val offsetStore = new JavadslCassandraOffsetStore(system, testSession, testCasConfigProvider, ReadSideConfig())
+
+  "ReadSide" must {
+    "not auto create offset store table when 'lagom.persistence.read-side.cassandra.tables-autocreate' flag is 'false'" in {
+      offsetStore.startupTask.isCompleted shouldBe true
+    }
```

Comment:
Ah, that wrong description :D I think it should be something like:
"ClusterStartupTask message should not be sent and future should be returned with Done result immediately"

What do you think?